<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDashboard</class>
 <widget class="QDialog" name="MainDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>720</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="SearchBar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:6px;
paddding-top:4px;
padding-bottom:4px</string>
           </property>
           <property name="placeholderText">
            <string>Find a task in any project</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SearchBtn">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(5, 109, 120);
color: rgb(255, 255, 255);
border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;

</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ProfileBtn">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(227, 0, 114);
color: rgb(255, 255, 255);

border-radius:8px;
padding-top:4px;
padding-bottom:4px;
padding-left:25px;
padding-right:25px;</string>
           </property>
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="sidebar">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="sidebarLayout">
                <item>
                 <widget class="QLabel" name="BoardTitle">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Board Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="SearchCurrentBoard">
                  <property name="maximumSize">
                   <size>
                    <width>1200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-radius:4px;

paddding-top:4px;
padding-bottom:4px;</string>
                  </property>
                  <property name="placeholderText">
                   <string>Enter task title...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="ProjectSettings">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(47, 64, 81);
color: rgb(255, 255, 255);

border-radius:8px;
padding:4px</string>
                  </property>
                  <property name="text">
                   <string>Project Settings</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="ProjectTimelineBtn">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(47, 64, 81);
color: rgb(255, 255, 255);

border-radius:8px;
padding:4px</string>
                  </property>
                  <property name="text">
                   <string>Project Timeline</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="OtherProjectsBtn">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(47, 64, 81);
color: rgb(255, 255, 255);

border-radius:8px;
padding:4px</string>
                  </property>
                  <property name="text">
                   <string> Other Projects</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="ToDo">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>To Do</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="TaskList">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>500</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-radius:6px;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="InProgress">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>In Progress</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="InProgressList">
                 <property name="styleSheet">
                  <string notr="true">border-radius:6px;
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="Review">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Review</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="ReviewList">
                 <property name="styleSheet">
                  <string notr="true">border-radius:6px;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="Done">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Done</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="DoneList">
                 <property name="styleSheet">
                  <string notr="true">border-radius:6px;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
